policy/common/methods.go

```diff
@@ -119,5 +131,15 @@ func (m *Methods) CommentMatches(commentBody string) bool {
 			return true
 		}
 	}
+	for _, comment := range m.GithubReviewComments {
+		if strings.Contains(commentBody, comment) {
+			return true
+		}
+	}
+	for _, pattern := range m.GithubReviewCommentPatterns {
+		if pattern.Matches(commentBody) {
+			return true
+		}
+	}
```

Comment:
I believe this will have to be in a new method so that review comment patterns are not matched against regular comment patterns and vice versa.